linux/LinuxProcess.c

```diff
@@ -159,6 +161,40 @@ bool LinuxProcess_setIOPriority(Process* this, Arg ioprio) {
    return (LinuxProcess_updateIOPriority((LinuxProcess*)this) == ioprio.i);
 }
 
+bool LinuxProcess_isAutogroupEnabled(void) {
+   FILE* file = fopen(PROCDIR "/sys/kernel/sched_autogroup_enabled", "r");
+   if (!file)
+      return false;
+   unsigned int enabled;
+   int count = fscanf(file, "%u", &enabled);
+   fclose(file);
```

Comment:
```suggestion
   char buf[16];
   if (xReadfile(PROCDIR "/sys/kernel/sched_autogroup_enabled", buf, sizeof(buf)) < 0)
      return false;

   unsigned int enabled;
   int count = sscanf(buf, "%u", &enabled);
```

Reply:
Then we would not even need the `scanf()`, as we could just check `buf[0]` (which should be either '0' or '1').